.github/actions/templates/getParameterFiles/action.yml

```
@@ -0,0 +1,34 @@
+name: 'Get parameter files'
+description: 'Retrieve the parameter file paths of a given module'
+
+inputs:
+  modulePath:
+    description: "The path to the module's folder"
+    required: true
+
+outputs:
+  parameterFilePaths:
+    description: 'The parameter files to use for template evaluation'
+    value: ${{ steps.get-parameter-file-paths.outputs.parameterFilePaths }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: 'Get parameter file paths'
+      id: get-parameter-file-paths
+      shell: pwsh
+      run: |
+        # Grouping task logs
+        Write-Output "::group::Get parameter files"
+        # Load used functions
+        . (Join-Path $env:GITHUB_WORKSPACE 'utilities' 'pipelines' 'sharedScripts' 'Get-ModuleParameterFiles.ps1')
+        $functionInput = @{
+          ModulePath = Join-Path $env:GITHUB_WORKSPACE '${{ inputs.modulePath }}'
+        }
+        Write-Verbose "Invoke task with" -Verbose
+        Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
+        # Set agent up
```

Probably copy pasted from another action, unless we need specific PS modules for this one. 
```suggestion
        # Get the list of parameter file paths
```